support-frontend/assets/components/directDebit/directDebitForm/directDebitForm.tsx

```diff
@@ -111,12 +122,16 @@ export default function DirectDebitForm(props: PropTypes): JSX.Element {
 			/>
 
 			{formError && (
-				<div className="component-direct-debit-form__error-message-container">
+				<div
+					css={css`
+						margin-top: 8px;
+					`}
+				>
 					<ErrorMessage message={formError} svg={<SvgExclamationAlternate />} />
 				</div>
 			)}
 
-			<PaymentButton onConfirmClick={onSubmit} />
+			{/* <PaymentButton onConfirmClick={onSubmit} /> */}
```

Comment:
Removing this because once we’re no longer popping up the modal we won’t need it, I assume. Why comment out rather than deleting?

Reply:
Yeah that's right, we wouldn't need the component but the onSubmit handler will probably be moved around slightly. The reason for commenting out is that there's another card in play that will be touching the same code and most likely moving around that button handler. Whilst conflicts can't be completely avoided in this case I thought commenting out may potentially make life easier

Reply:
Ah yeah, that makes sense :)